exporters/otlp/mock_collector_test.go

```diff
@@ -207,15 +207,15 @@ func (mc *mockCol) getMetrics() []*metricpb.Metric {
 	return mc.metricSvc.getMetrics()
 }
 
-// runMockCol is a helper function to create a mockCol
-func runMockCol(t *testing.T) *mockCol {
-	return runMockColAtAddr(t, "localhost:0")
+// runMockCollector is a helper function to create a mock Collector
+func runMockCollector(t *testing.T) *mockCol {
+	return runMockColAtEndpoint(t, "localhost:0")
 }
 
-func runMockColAtAddr(t *testing.T, addr string) *mockCol {
-	ln, err := net.Listen("tcp", addr)
+func runMockColAtEndpoint(t *testing.T, endpoint string) *mockCol {
```

Comment:
nitpick: if we are going to switch to the full word collector as the above test, probably a good idea to consistently do it.

```suggestion
func runMockCollectorAtEndpoint(t *testing.T, endpoint string) *mockCol {
```